ui/ts/models/log.ts

```diff
@@ -58,39 +58,58 @@ module Models {
         true
       );
 
-      private _url(): string {
-        let url: string = "/_status/logs/";
+      /**
+       * buildURL creates the route and query parameters based on the current
+       * state of all the properties on entries.
+       * TODO(bram): consider exporting this to a utility function and maybe
+       * adding an interface for these top level models. Tamir also suggested
+       * moving to a single param with a tighter serialized format.
+       */
+      buildURL(): string {
+        let url: string = "/logs/";
```

Comment:
It actually looks like you could use m.route(url, struct), where struct is a dictionary of parameters.

Honestly, even if it was just thrown in there by the mithril guys, it's still a better choice than manual construction like this.